src-docs/src/views/modal/modal_example.js

```diff
@@ -41,17 +41,17 @@ const modalSnippet = `<EuiModal onClose={this.closeModal}>
 const confirmModalSnippet = [
   `<EuiConfirmModal
   title="Modal title goes here"
```

Comment:
```suggestion
  title={title}
```
I'd make this a variable as the type of this prop is ReactNode but thinking some more about it I don't see why this is node and not just a string... I guess that's a separate discussion.